### PR TITLE
initial commit of all base files to repository

### PR DESCRIPTION
**Pull request by twsleight**
This includes 1 .py file and several data files.

### CODE REVIEW

File: benzene_gas.out [hunk 1]
```diff
@@ -0,0 +1,14401 @@
+ Entering Gaussian System, Link 0=g09
```
Comment (paulboone) [inline benzene_gas.out]: Are these .out files required to run the software, or are they just what you are using for your problem? If they are not required for everybody, then they either shouldn't be included, or could be included as an example problem for other people to get started with.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,86 @@
+attrs==19.3.0
```
Comment (paulboone) [inline requirements.txt:1]: In a future PR, you should replace this with the actual software requirements, not the full pip freeze list.

**Review by paulboone — COMMENTED** [review]
Before I merge this, let me know about the .out files and if you want them in the repo as example files, or keep them outside. (we can talk about this too).

File: benzene_gas.out (hunk 1 above)
Comment (twsleight) [reply benzene_gas.out]: The .out files are what this software package analyzes. I guess they're not technically required, but you can't really do anything with the software without them. I can move them to an example. thanks!

File: tests/readtest.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+#should be the final structure in the gaussian .out file.
+#coumarin.out files are used for testing
+import glob
+namelist=[]
+
+for filename in glob.glob('*.out'):
+    print(filename)
+    namelist.append(filename)
+        with open(filename)as searchfile:
+            pass
```
Comment (paulboone) [inline tests/readtest.py:14]: This file should be split into a separate commit. Also looks like this is the start to something, so might not be ready to merge.

**Review by paulboone — CHANGES_REQUESTED** [review]
When you move files in git, you will want to add them to the new location (i.e. examples) and delete them from the old location in the same commit. Otherwise it looks like you are adding duplicate files unnecessarily. You can fix this here by just squashing the commits where you remove files with the ones where you add them.
